cdap-app-fabric/src/test/java/io/cdap/cdap/internal/app/services/SourceControlManagementServiceTest.java

```diff
@@ -373,7 +379,7 @@ public void testPullAndDeployAuthenticationConfigException() throws Exception {
     sourceControlService.setRepository(namespaceId, REPOSITORY_CONFIG);
 
     Mockito.doThrow(new AuthenticationConfigException("Repo config is invalid"))
-      .when(sourceControlOperationRunnerSpy).pull(Mockito.any());
+        .when(sourceControlOperationRunnerSpy).pull(Mockito.any(PulAppOperationRequest.class));
```

Comment:
typo: Pul -> Pull (can do this in a separate PR)

Reply:
Done